src/main/kotlin/com/chimbori/crux/api/Fields.kt

```diff
@@ -8,6 +8,8 @@ public object Fields {
   public const val LANGUAGE: String = "language"
   public const val DISPLAY: String = "display"
   public const val ORIENTATION: String = "orientation"
+  public const val CREATED: String = "created"
```

Comment:
How about calling this `PUBLISHED_AT` instead of `CREATED`, so it uses the standardized name from OpenGraph? In the context of articles, `published` seems more appropriate than `created`. I am guessing that Schema.org is way more generic, so they went with a `dateCreated`.

Adding the `_AT` suffix makes it clear to callers what the format is, and removes the distinction between `Date` (without time) and `DateTime` since ISO 8601 defines both in the same format.

`CREATED_AT` would also be fine, WDYT?

Reply:
Agree with the naming change. I had arbitrarily chosen between schema.org's and opengraph's convention. I like your reasoning for the names. Will update.